tests/para-tests-no-ci/test-xcm-para.ts

```diff
@@ -175,6 +208,38 @@ describeParachain("XCM - send_relay_asset_to_relay", { chain: "moonbase-local" }
     ({ assetId } = await registerAssetToParachain(parachainOne, alith));
 
     // RELAYCHAIN
+    // Set supported version
+    // Release-v0.9.12 does not have yet automatic versioning
+    await createBlockWithExtrinsicParachain(
+      relayOne,
+      aliceRelay,
+      relayOne.tx.sudo.sudo(relayOne.tx.xcmPallet.forceDefaultXcmVersion(1))
+    );
+
+    let beforeAliceRelayBalance = ((await relayOne.query.system.account(aliceRelay.address)) as any)
+      .data.free;
+    // Fees
+    const fee = (
+      await relayOne.tx.xcmPallet
+        .reserveTransferAssets(
```

Comment:
this call could be stored in a variable and resued in l247 (also makes sure that we are using the same call)